python/test/t_DirectionalSampling_std.py

```
@@ -79,3 +79,22 @@
+    
+# Test setter of event
+ot.RandomGenerator.SetSeed(0)
+X = ot.RandomVector(ot.Normal(2))
+Y = ot.CompositeRandomVector(ot.SymbolicFunction(['X1','X2'], ['X1']),X)
+event = ot.ThresholdEvent(Y, ot.Less(), -2.0)
+algo = ot.DirectionalSampling(event)
+algo.setMaximumOuterSampling(100000)
+algo.setMaximumCoefficientOfVariation(0.01)
+algo.run()
+result = algo.getResult()
+assert_almost_equal(result.getProbabilityEstimate(), ot.Normal().computeCDF(-2), 1.0e-1, 0.0)
```

Ruff (0.15.15)

[error] Undefined name `assert_almost_equal`

(F821)